### DISCUSSION TITLE
Using a Video/GIF as a Texture

### DISCUSSION THREAD

**Discussion by ebbicott** [bnpr/Malt#399] [Q&A] [answered]
Using sampler I can easily use an image as a 2d texture (for a computer screen) and it looks great with everything else. Is there a way to load a Video or at least series of images as a texture for an animation? There's the standard Image Texture node that supports it so I'm looking for something like that.

**Comment by pragma37** [comment, marked as answer]
This is actually a limitation on the Blender side: https://developer.blender.org/T37216

If you can fit all your frames into a flipbook texture you can use the Flipbook and the Time Info nodes.

**Reply by ebbicott** [reply]
I'm sorry if I'm missing something obvious but I can't find the "Flipbook" node anywhere in the Malt Node Tree editor. I see it in the documentation but there's not too much information there in how to use it.

**Reply by Kolupsy** [reply]
Depending on what version of malt you are using, it should be available in the Texturing category. It is very easy to use. Simply load your flipbook texture into the node, define the dimensions of the grid of your flipbook and then all thats left to do is animating the 'Page' parameter. 

![image](https://user-images.githubusercontent.com/88381426/192461199-a6fff9c5-3884-4cea-92b9-36fc9d71d75b.png)

**Reply by ebbicott** [reply]
It was the version of malt, I installed the "node-groups-latest" and it's there. I think I was using "Release-latest" which didn't have all that. Thank you!

**Comment by ebbicott** [comment]
Okay, I'll try that out. Thanks.
